components/fields/datePicker/DatePickerField.tsx

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+// Copyright (c) 2021. Sendanor <info@sendanor.fi>. All rights reserved.
+
+import React, { ReactNode, useState } from 'react';
```

Comment:
We do not use default imports since they are problematic. Refactor code that you find so that it doesn't use them, if it's possible. (For example, `moment` isn't, so it is a special case.) 

However, when you see `React.Foo`, refactor to import it as named `{ Foo }` instead.

This will make it possible to use the code with multiple different build systems. For example, building for frontend, CLI, backend and SSR Server will require multiple different build systems that are not compatible in features with default imports.

It will also make the built bundle smaller.